RELEASE_NOTES.md

```diff
@@ -1,42 +1,279 @@
+
+For customers current running Chef Infra Client as a service we suggest migrating to scheduled task based execution. This allows for complex scheduling scenarios not possible with simple services such, as the ability to skip Chef Infra Client execution if a system is on battery or to run the Chef Infra Client as soon as a system boots to ensure configuration.
+
+Chef Infra Client can be configured to run as a scheduled task using the [chef-client cookbook](https://supermarket.chef.io/cookbooks/chef-client) or ideally using the [chef_client_scheduled_task resource](https://docs.chef.io/resources/chef_client_scheduled_task/) built into Chef Infra Client 16 or later. For users already running as a service setting up the scheduled task and then stopping the existing service can be performed within a Chef Infra Client run to migrate systems.
+
+#### Gem Resource Ruby 1.9+
+
+The `gem` resource used to install Ruby Gems into the system's Ruby installation will now assume Ruby 1.9 or later. As Ruby 1.8 or below have been end of life for nearly 7 years we believe there is little to no impact in this change.
+
+#### Legacy node['filesystem2'] removed on AIX/Solaris/FreeBSD
+
+The legacy `node['filesystem2']` attributes leftover from our multi-year migration of filesystem data on AIX, Solaris, and FreeBSD systems has been removed. This same data is now available at `node['filesystem']`
+
+#### Removed Antergos and Pidora Detection
+
+Ohai detection of the end of life Antergos and Pidora distributions has been removed. Antergos ended releases and downloads of the distribution in May 2019 and Pidora stops receiving updates in 2014.
 
 ### Infra Language Improvements
 
-- New effortless? helper - https://github.com/chef/chef/pull/11150
-- Default values in custom resources are now dup'd - https://github.com/chef/chef/pull/11095
-- Lazy attribute loading: https://github.com/chef/chef/pull/10861
-- reboot_pending? now works on all debian platform_family distros not just Ubuntu specifically - https://github.com/chef/chef/pull/10989
+#### Lazy Attribute Loading
+
+A common problem when using the "wrapper cookbook" pattern is when the wrapped cookbook declares what are called "derived attributes", which are attributes that refer to other attributes. Because of the order that attribute files are parsed in, this does not work as intended when the base attribute is changed in a wrapper cookbook. By extending the use of the `lazy {}` helper to the declaration of node attributes, it makes it possible for the wrapped cookbook to cleanly allow wrapper cookbooks to override base attributes as intended.
+
+Instead of:
+
+```ruby
+default['myapp']['dir'] = '/opt/myapp'
+default['myapp']['bindir'] = "#{node['myapp']['dir']}/bin"
+```
+
+Use the lazy helper:
```

Comment:
Move the lazy helper example above the less-desired pattern. Show people what you want them to do before showing them what you don't want them to do.

Reply:
Done